core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlExecuteStatement.java

```diff
@@ -55,12 +55,12 @@ protected boolean executeAction() throws InterruptedException {
         String sqlStatement = convertSqlStatement(getSqlStatement().getValue());
         String connectionName = convertConnectionName(getConnectionName().getValue());
         // Get Connection
-        Connection connection = ConnectionConfiguration.getInstance().get(new ConnectionKey(connectionName, getExecutionControl().getEnvName()))
-                .orElseThrow(() -> new RuntimeException("Cannot find connection " + connectionName));
+        Connection connection;
+        connection = ConnectionConfiguration.getInstance()
+                .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
+                .orElseThrow(() -> new RuntimeException("Unknown connection name"));
```

Comment:
add 'connectionName' to the Exception message: cannot find connection 'connectionName'. This will help users during debugging

Reply:
done in next commit